### DISCUSSION TITLE
Fiz uma versão em PT-BR do curso. I made a PT-BR version of the course.

### DISCUSSION THREAD

**Discussion by ghost** [skills/.github#333] [Introduction to GitHub] [answered]
O link para os arquivos traduzidos é [Introdução ao GitHub](https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-srcbug/tree/main). Cada passo possui sua tradução em um arquivo separado. Não sei se já existe uma versão PT-BR oficial do curso, para mim estava todo em inglês, então decidi criar uma versão PT-BR e compartilhar com as pessoas que necessitam de uma versão traduzida.

The link to the translated files is [Introdução ao GitHub](https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-srcbug/tree/main). Each step has its translation into a separate file. I don't know if there is already an official PT-BR version of the course, for me it was all in english, so I decided to create a PT-BR version and share with the people who need a translated version.

**Comment by sinsukehlab** [comment]
Why did you put the translated files in the root (`/`) directory? You might want to put them in the `.github/steps` directory.
Suggested file names: 0-bem-vindo.md, 1-crie-uma-ramificação.md, 2-enviar-um-arquivo.md, 3-abrir-uma-solicitação-de-mescla.md, 4-mesclar-sua-solicitação-de-pull.md, X-fim.md

**Reply by ghost** [reply]
Excellent idea! Now each step will update as in the official. Thank you!

**Reply by sinsukehlab** [reply]
You're welcome.
Also, you might want to make the repository template so that you can generate another repository.
You can generate a repository from the [skills official template](https://github.com/skills/introduction-to-github) in order to remove GitHub Classroom artifacts.

**Reply by ghost** [reply]
Like this? [introducao-ao-github](https://github.com/srcbug/introducao-ao-github) Again, **thanks for your time!  :D**

**Reply by sinsukehlab** [reply, marked as answer]
LGTM! I'm glad that I could be of help.
